### ISSUE TITLE
Create first CNN to train and test light source recognition

### ISSUE THREAD

**Issue by NemoHostem** [NemoHostem/CNN_Source_of_Light#13]
**Description:**
Create first CNN to train and test light source recognition for faces only

**Tasks:**
- [x] Create a basic Keras / Tensorflow network (5 layers)
- [x] Do not split data into smaller chunks before input
- [x] Measure performance (network creation speed, test speed) and accuracy (how many correct)
- [x] Test with generated data
- [x] Test with real world data (faces)
